react/features/base/themes/functions.web.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+
+import { createMuiTheme } from '@material-ui/core/styles';
+
+import { createColorTokens } from '../ui/functions';
+
+/**
+ * Creates a MUI theme based on local UI tokens.
+ *
+ * @param {Object} arg - The ui tokens.
+ * @returns {Object}
+ */
+export function createTheme({ font, colors, colorMap, shape, spacing, typography }: Object) {
```

Comment:
This doesn't need to be exported, does it?

Reply:
Yes it does. I have renamed it to be more explicit.